.github/workflows/e2e.yaml

```diff
@@ -0,0 +1,25 @@
+on:
+  pull_request_target:
+    types: [labeled]
+name: e2e
+jobs:
+  integration:
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'ok-to-test')
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+          repository: kubevirt/project-infra
+          path: project-infra
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
```

Comment:
Will be interesting to see if github actions will be used here too. We potentially have to delete them.